java/test/src/main/java/org/ray/api/TestUtils.java

```diff
@@ -28,8 +28,13 @@ public static void skipTestUnderSingleProcess() {
   }
 
   public static void skipTestIfDirectActorCallEnabled() {
-    if (ActorCreationOptions.DEFAULT_USE_DIRECT_CALL) {
-      throw new SkipException("This test doesn't work when direct actor call is enabled.");
+    skipTestIfDirectActorCallEnabled(true);
+  }
+
+  public static void skipTestIfDirectActorCallEnabled(boolean enabled) {
```

Comment:
Maybe this is more clear?

```suggestion
  public static void skipTestIfDirectActorCallDisabled() {
```